CODA/ReadMe.txt

```diff
@@ -0,0 +1,30 @@
+   ______          __     
+  / ____/___  ____/ /___ _
+ / /   / __ \/ __  / __ `/
+/ /___/ /_/ / /_/ / /_/ / 
+\____/\____/\__,_/\__,_/  
+                          
+
+A smoothswing font, copyright (C) Greyscale Fonts 2022
```

Comment:
So, it's a little weird to me that this copy of the font has a different readme.txt file than the one distributed on greyscalefonts.com. I have no way to verify what we're allowed to do with this font because of that.

Reply:
I reached out to Sam and he's putting the same versions of the ReadMe and the license on his site's version for download.

Reply:
His copy now has a license.txt (great!)
This copy should also have it.

Reply:
> His copy now has a license.txt (great!) This copy should also have it.

It does though. Right after in04.wav

Reply:
Ah, I missed that. (because the diff didn't load by default.)
I haven't listened to this font, I'm going to assume that the reason you submitted a PR for it is because it's a good general-purpose font that belongs on a "default sd card" kind of place?

Reply:
Yes. You should try it with that nice large speaker :) It's smooth, plays very well on a saber, and primarily because it features more modern ProffieOS supported sounds, like preon, accent swings, bgn/end sounds.
I'd like to get one in that is complete, with a melt, lb, stab etc...